### ISSUE TITLE
SPM Build Failure

### ISSUE THREAD

**Issue by peymankh** [yonaskolb/XcodeGen#821]
Getting build failures with SPM on the latest version using Xcode 11.4.0.

```
[17:20:12]: Exit status of command 'mint bootstrap' was 1 instead of 0.
🌱 Cloning XcodeGen 2.15.0
🌱 Resolving package
error: because no versions of XcodeProj match the requirement 7.8.0 and root depends on XcodeProj 7.8.0, version solving failed.
🌱 Encountered error during "swift package resolve". Use --verbose to see full output
🌱  Failed to resolve XcodeGen 2.15.0 with SPM
```

**Comment by peymankh** [comment]
Seems Xcodeproj have remove version `7.8.0`

https://github.com/tuist/XcodeProj/releases

**Comment by yonaskolb** [comment]
Fixed in 2.15.1
